src/server/scripts/EasternKingdoms/ScarletEnclave/chapter1.cpp

```diff
@@ -865,87 +824,21 @@ class spell_stable_master_repo : public AuraScript
     }
 };
 
-class spell_deliver_stolen_horse : public SpellScript
+class spell_death_race_complete : public SpellScript
 {
-    PrepareSpellScript(spell_deliver_stolen_horse);
+    PrepareSpellScript(spell_death_race_complete);
 
-    bool Validate(SpellInfo const* /*spellInfo*/) override
+    void HandleScript(SpellEffIndex /*effIndex*/)
     {
-        return ValidateSpellInfo({ SPELL_DELIVER_STOLEN_HORSE, SPELL_EFFECT_STOLEN_HORSE });
-    }
-
-    void HandleScriptEffect(SpellEffIndex /*effIndex*/)
-    {
-        Unit* target = GetHitUnit();
-        target->RemoveAurasDueToSpell(SPELL_EFFECT_STOLEN_HORSE);
-
-        Unit* caster = GetCaster();
-        caster->RemoveAurasDueToSpell(SPELL_EFFECT_STOLEN_HORSE);
-        caster->RemoveFlag(UNIT_NPC_FLAGS, UNIT_NPC_FLAG_SPELLCLICK);
-        caster->SetFaction(FACTION_FRIENDLY);
-
-        caster->CastSpell(caster, SPELL_CALL_DARK_RIDER, true);
+        GetHitUnit()->ExitVehicle();
```

Comment:
After removing aura `52349` camera is getting stuck in place (character moves though). Could someone please help with this issue? Currently have no idea.

Reply:
Maybe I have misunderstood something in these changes, but while viewing the code in split screen, it looks like aura 52349 is not handled by any enum or reference. Could that be part of the issue?

Reply:
It's not used in C++ code (only applied in SQL).

Reply:
Fair enough, I misunderstood how it is handled.